cmd/abci-cli/abci-cli.go

```diff
@@ -259,7 +259,7 @@ func cmdBatch(cmd *cobra.Command, args []string) {
 func cmdConsole(cmd *cobra.Command, args []string) {
 
 	for {
-		fmt.Printf("\n> ")
```

Comment:
the extra line is not necessary IMO and makes for a less nice experience because we end up with double line breaks